test/modules/print/DailyPrintView.test.js

```diff
@@ -230,14 +230,14 @@ describe('DailyPrintView', () => {
       Renderer.data.bgRange[1] = 100;
       Renderer.makeScales(Renderer.chartsByDate[sampleDate]);
 
-      expect(Renderer.bgScaleYLimit).to.equal(180);
+      expect(Renderer.bgScaleYLimit).to.equal(300);
```

Comment:
The test description should be changed if we are now setting the `bgScaleYLimit` to a minimum of the very high threshold.